manim/mobject/three_dimensions.py

```diff
@@ -340,7 +340,7 @@ def construct(self):
     """
 
     def __init__(self, point=ORIGIN, radius=DEFAULT_DOT_RADIUS, color=WHITE, **kwargs):
-        Sphere.__init__(self, center=point, radius=radius, **kwargs)
+        Sphere.__init__(self, center=point, radius=radius, resolution=(8, 8), **kwargs)
```

Comment:
This will cause an error to be thrown if a user passes in a resolution parameter:
```py
TypeError: manim.mobject.three_dimensions.Sphere.__init__() got multiple values for keyword argument 'resolution'
```
a safer approach would be to include it in `__init__` and doing `resolution=resolution`

(also change a `Sphere.__init__` to super since you're already on that line 😄)
```suggestion
        super().__init__(center=point, radius=radius, resolution=(8, 8), **kwargs)
```

Reply:
Noted, thanks for the quick response!
But shouldn't I change the ```resolution``` to ```(8,8)``` in the ```__init__`` method ?

Reply:
Yes, and do `resolution=resolution` in the superclass init.